src/components/EtherscanLink/style.js

```diff
@@ -1,11 +1,30 @@
 // @flow
+import { secondaryText } from '~/theme/variables'
 
 export const styles = () => ({
   etherscanLink: {
     display: 'flex',
     alignItems: 'center',
+
```

Comment:
why empty line?

Reply:
Readability I guess.

As it's common to use in sass/scss nesting: https://sass-lang.com/guide#topic-3